omod/src/main/java/org/openmrs/module/smartonfhir/web/filter/SmartSessionLogoutFilter.java

```diff
@@ -36,13 +36,13 @@
 	@Override
 	public void init(FilterConfig filterConfig) {
 		final KeycloakUriBuilder keycloakUriBuilder = KeycloakDeploymentBuilder
-				.build(KeycloakConfigHolder.getKeycloakConfig()).getLogoutUrl();
-
+		        .build(KeycloakConfigHolder.getKeycloakConfig()).getLogoutUrl();
+		
 		if (keycloakUriBuilder != null) {
 			logoutUrl = keycloakUriBuilder.toTemplate();
 		} else {
 			log.error(
-					"Could not find Keycloak configuration file. Please run keycloak server before openmrs to avoid this error");
+			    "Could not find Keycloak configuration file. Please run keycloak server before openmrs to avoid this error");
```

Comment:
do we need changes in this file?

Reply:
yes @HerbertYiga, these changes are generated after mvn clean install.